### DISCUSSION TITLE
Adding Options directly from form.

### DISCUSSION THREAD

**Discussion by ershadow786** [area17/twill#864] [Q&A]
Is there any about this https://github.com/area17/twill/blob/9e9c7f8d1f9931fc47451576d921859c289976a1/views/partials/form/_select.blade.php#L35

Can we use this option now? I have been using this option but problem i am facing is 

1. After adding new option it doesn't populate new value to dropdown. I have refresh page for changes,can anyone help me this,how can i achieve this?

2. I want to do same thing with input field also, so anyone can point me in right direction how can i able to do it?

**Comment by ershadow786** [comment]
Can anyone help me with this?

**Comment by Inxo** [comment]
https://github.com/area17/twill/blob/9e9c7f8d1f9931fc47451576d921859c289976a1/views/partials/form/_select.blade.php#L15
